api/tests/test_hyp3_api.py

```diff
@@ -2,11 +2,13 @@
 from json import dumps
 from os import environ
 
+import boto3
+import hyp3_api
 import pytest
 from botocore.stub import Stubber
 from flask_api import status
 from hyp3_api import STEP_FUNCTION_CLIENT, auth, connexion_app
```

Comment:
not in love with `import hyp3_api` and `from hyp3_api import ....`, probably just to the one full import and change the three places auth/STEP_FUNCTION_CLIENT/connexion_app are referenced